[22634] Fix scaling of mobilePick menus when device is on landscape

livecodepanos opened the pull request:
This patch ensures mobilePick and mobilePickDate menus use the full width of the screen when the device is on landscape orientation.

Closes https://quality.livecode.com/show_bug.cgi?id=22634

livecodepanos commented:
Before:

<img width="779" alt="Screenshot 2020-03-31 at 18 29 33" src="https://user-images.githubusercontent.com/5111835/78117790-c1eb2a00-740e-11ea-834b-d7d2154e0f64.png">

<img width="779" alt="Screenshot 2020-04-01 at 11 20 21" src="https://user-images.githubusercontent.com/5111835/78117827-cb749200-740e-11ea-83b0-e49f33fa0ebe.png">

After:

<img width="779" alt="Screenshot 2020-03-31 at 18 28 19" src="https://user-images.githubusercontent.com/5111835/78117855-d4656380-740e-11ea-8827-7fadd894ebda.png">

<img width="779" alt="Screenshot 2020-04-01 at 11 21 56" src="https://user-images.githubusercontent.com/5111835/78117871-da5b4480-740e-11ea-908a-5b933f40de8b.png">

montegoulding reviewed (changes requested):
@livecodepanos if we are touching these lines we should probably re-write sans ternary operator to be inline with coding conventions

livecodepanos committed:
[22634] Fix scaling of mobilePick menus when device is on landscape

This patch ensures mobilePick and mobilePickDate menus use the full width
of the screen when the device is on landscape orientation.

montegoulding commented:
@livecode-vulcan review ok 508406d

livecode-vulcan commented:
💙 review by @montegoulding ok 508406dc1edc0b5166f83433f6175492b454d6a2

livecode-vulcan commented:
😎 test success 508406dc1edc0b5166f83433f6175492b454d6a2

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/748)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/157)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/157)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/157)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/157)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2274)
* try-community-universal-ios-iphoneos13.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.4/builds/8)
* try-community-universal-ios-iphonesimulator13.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.4/builds/8)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2303)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2290)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2289)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2324)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2264)
